funds-manager/funds-manager-server/src/execution_client/venues/quote.rs

```diff
@@ -0,0 +1,166 @@
+//! Type definitions for execution quotes
+
+use alloy_primitives::U256;
+use funds_manager_api::{quoters::ApiExecutionQuote, u256_try_into_u128};
+use renegade_common::types::{chain::Chain, token::Token};
+
+use crate::execution_client::{
+    error::ExecutionClientError,
+    venues::{lifi::LifiQuoteExecutionData, SupportedExecutionVenue},
+};
+
+/// The basic information included in an execution quote,
+/// agnostic of the venue that provided the quote
+pub struct ExecutionQuote {
+    /// The token being sold
+    pub sell_token: Token,
+    /// The token being bought
+    pub buy_token: Token,
+    /// The amount of the token being sold, in atoms
+    pub sell_amount: U256,
+    /// The quoted amount of the token being bought, in atoms
+    pub buy_amount: U256,
+    /// The venue that provided the quote
+    pub venue: SupportedExecutionVenue,
+    /// The chain for which the quote was generated
+    pub chain: Chain,
+}
+
+impl ExecutionQuote {
```

Comment:
other than this `impl` block, the code in this file is just me moving the types over from `execution_client/venues/mod.rs`